libs/editor/WordPressEditor/build.gradle

```diff
@@ -91,6 +86,15 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.6.2'
 
     lintChecks 'org.wordpress:lint:1.0.1'
+
+    // We need to add the hermes library manually for binary dependency.
+    // There doesn't seem to be a good way to integrate an aar into a library dependency, and it's
+    // not something Google wants to support, so this is a necessary workaround.
```

Comment:
Here is a Stackoverflow answer by "Xavier Ducrohet" who is the "Android SDK Tech Lead - Google Inc." according to their profile: https://stackoverflow.com/a/20715155

@mokagio @mchowning I don't typically like to refer to stackoverflow from code comments, but I guess this one is OK since it's from an official source. What do you think?

I am also open to the idea of rewording this comment if you can think of a better way to explain this.

Reply:
>  I don't typically like to refer to stackoverflow from code comments, but I guess this one is OK since it's from an official source. What do you think?

I think that's okay in this instance. Even if the comment was reworded, I'd still look at it and ask "what's the source?". I would have been nice if Google had an official page about this but 🤷‍♂️ 


P.S. _That comment is from 2013_ 😱 They really must not care about this kind of problem / don't see it as the right problem to tackle.

Reply:
I've changed the wording a bit and added the link in 47aa161bb56d75a957dcac4b656870980b162629.

> P.S. That comment is from 2013 😱 They really must not care about this kind of problem / don't see it as the right problem to tackle.

I think it might just be due to its complication. They might believe that it's best to publish each component separately. I haven't given it enough thought, but I think there is merit to it. I'll see if we can upload this binary somewhere and read it from there, but I don't want it blocking the merge.